tardis/tests/tests_slow/test_w7.py

```diff
@@ -39,11 +37,66 @@ class TestW7(object):
         * last_line_interaction_shell_id |     * montecarlo_nu
         * nubar_estimators               |     * last_line_interaction_angstrom
         * ws                             |     * j_blues_norm_factor
+
+    * spectrum.npz
+    Contents (all (.npy)):
+        * luminosity_density_nu          | * wavelength
+        * delta_frequency                | * luminosity_density_lambda
+    """
+
+    # TODO: make this fixture ingest data from an HDF5 file.
+    slow_test_data_dir = os.path.join(os.path.expanduser(
+            os.path.expandvars(request.config.getvalue('slow-test-data'))),
+            "w7")
+
+    ndarrays = dict(np.load(os.path.join(slow_test_data_dir, "ndarrays.npz")))
+    quantities = dict(np.load(os.path.join(slow_test_data_dir, "quantities.npz")))
+    spectrum = dict(np.load(os.path.join(slow_test_data_dir, "spectrum.npz")))
+
+    # Associate CGS units to ndarrays of baseline quantities.
+    quantities['j_blues_norm_factor'] = \
+        quantities['j_blues_norm_factor'] * u.Unit('1 / (cm2 s)')
```

Comment:
As I wrote in the other comment, use `units.Quantity(quantities['j_blues_norm_factor'], '1 / (cm2 s)')`.